Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionCocoa.mm

```diff
@@ -237,6 +233,11 @@
         return false;
     }
 
+    auto manifestJSONString = String([[NSString alloc] initWithData:manifestData encoding:NSUTF8StringEncoding]);
+    m_manifestJSON = JSON::Value::parseJSON(manifestJSONString);
+    if (!m_manifestJSON)
```

Comment:
`m_manifestJSON` is a `Ref`, so the result of `JSON::Value::parseJSON()` must be checked for `nullptr` first. If parsing succeeds, assign `*result` to `m_manifestJSON`; otherwise, assign `JSON::Value::null()`. This mismatch is the cause of the build failure.

Reply:
You might want to move over `manifestParsedSuccessfully()` in this PR, and check `m_manifestJSON` for `isNull()`. Just for completeness on the parsing part.